@kmuehlbauer you were successful using `cmake` with `netcdf-cxx`. Any advice?

Linux:

``` shell
/opt/conda/envs/_build/bin/gfortran    -O2 -g -DNDEBUG CMakeFiles/f90tst_fill2.dir/f90tst_fill2.f90.o  -o f90tst_fill2  -L/feedstock_root/build_artefacts/work/netcdf-fortran-4.4.4/fortran  -L/feedstock_root/build_artefacts/work/netcdf-fortran-4.4.4/libsrc  ../fortran/libnetcdff.a /opt/conda/envs/_build/lib/libnetcdf.so.11.0.0 /opt/conda/envs/_build/lib/libmfhdf.so /opt/conda/envs/_build/lib/libdf.so /opt/conda/envs/_build/lib/libhdf5_hl.so /opt/conda/envs/_build/lib/libhdf5.so /usr/lib64/librt.so /opt/conda/envs/_build/lib/libz.so /usr/lib64/libdl.so /usr/lib64/libm.so /opt/conda/envs/_build/lib/libcurl.so -Wl,-rpath,/feedstock_root/build_artefacts/work/netcdf-fortran-4.4.4/fortran:/feedstock_root/build_artefacts/work/netcdf-fortran-4.4.4/libsrc 
/opt/rh/devtoolset-2/root/usr/bin/ld: warning: libjpeg.so.9, needed by /opt/conda/envs/_build/lib/libmfhdf.so, not found (try using -rpath or -rpath-link)
/opt/rh/devtoolset-2/root/usr/bin/ld: warning: libssl.so.1.0.0, needed by /opt/conda/envs/_build/lib/libcurl.so, not found (try using -rpath or -rpath-link)
/opt/rh/devtoolset-2/root/usr/bin/ld: warning: libcrypto.so.1.0.0, needed by /opt/conda/envs/_build/lib/libcurl.so, not found (try using -rpath or -rpath-link)
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_connect'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_set_defaults@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_check_issued'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_srp_password'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BIO_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BIO_s_mem'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_get_reader'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_get_string_type'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `sk_pop_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BIO_ctrl'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `DES_set_odd_parity'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_get0_user_data'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BN_num_bits'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_get_next'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `PEM_read_X509'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `MD4_Update'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_STRING_to_UTF8'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `GENERAL_NAMES_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `PKCS12_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get1_session'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_set_session'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_STRING_length'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_get_first'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_OpenSSL'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_get_closer'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_CreateCompress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OPENSSL_add_all_algorithms_noconf'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OPENSSL_load_builtin_modules'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_NAME_get_index_by_NID'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_verify'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_set_writer'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_SESSION_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `RAND_load_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `MD5_Init'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `sk_value'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_ctrl'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_load_error_strings'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_get_opener'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `MD5_Final'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_set_closer'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `CRYPTO_malloc'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_STORE_set_flags'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSLv2_client_method'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_set_quality@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `PKCS12_PBE_add'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_check_private_key'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_use_certificate_chain_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_BASICRESP_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_peer_certificate'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BIO_new'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_get_subject_name'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_RESPONSE_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_clear_error'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_NAME_get_entry'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `d2i_PKCS12_fp'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SHA256_Init'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `CONF_modules_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_basic_verify'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_cleanup'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BIO_puts'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `i2t_ASN1_OBJECT'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_alpn_protos'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_add_client_CA'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_certificate'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `RAND_egd'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_default_passwd_cb_userdata'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `CRYPTO_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSLv3_client_method'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_get_ext_d2i'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_resync_to_restart@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_destroy_compress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_free_strings'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `d2i_OCSP_RESPONSE'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_set_connect_state'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_new'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_response_status'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_verify_result'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_response_status_str'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_load_crl_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_create_method'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_read'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `RAND_bytes'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_set_fd'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `EVP_cleanup'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_use_PrivateKey'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_NAME_print_ex'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_finish_decompress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_pending'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_verify_cert_error_string'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_current_cipher'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_destroy_method'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_load_private_key'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_crl_reason_str'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_LOOKUP_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_finish'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `RAND_add'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_load_verify_locations'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CIPHER_get_name'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_STRING_type'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_write_scanlines@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_get_id'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_get_error'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_read_header@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `i2a_ASN1_OBJECT'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSLv23_client_method'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_INTEGER_get'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `sk_num'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_cert_status_str'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_STRING_data'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `EVP_PKEY_free'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_CreateDecompress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_privatekey'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_error_string'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `RAND_file_name'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_ctrl_cmd'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_peek_error'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_finish_compress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_use_PrivateKey_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SHA256_Update'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_library_init'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_check_validity'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_start_decompress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_set_opener'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_default_passwd_cb'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_use_certificate_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509V3_EXT_print'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `MD5_Update'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SHA256_Final'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_get_writer'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_get_pubkey'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_cipher_list'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BIO_printf'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_new'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_use_certificate'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_get_serialNumber'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_get_input_flags'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_write'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_resp_get0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_get_cert_store'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `i2d_X509_PUBKEY'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_start_compress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_response_get1_basic'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `CRYPTO_cleanup_all_ex_data'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_msg_callback'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `sk_pop'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_error_string_n'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `DES_set_key'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_shutdown'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_srp_username'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_TIME_print'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_version'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `DES_ecb_encrypt'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_destroy_decompress@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_shutdown'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_std_error@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_set_default'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_set_next_proto_select_cb'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_COMP_free_compression_methods'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_STORE_add_lookup'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_get_issuer_name'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_single_get0_status'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_EXTENSION_get_object'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `RAND_status'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `EVP_PKEY_copy_parameters'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_EXTENSION_get_data'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_load_builtin_engines'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `MD4_Final'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get0_alpn_selected'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_ctrl'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `BN_print'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_method_set_reader'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `MD4_Init'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `CONF_modules_load_file'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_ctrl'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ERR_remove_thread_state'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `X509_NAME_ENTRY_get_data'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_error'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `PEM_write_bio_X509'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_by_id'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_CTX_free'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `OCSP_resp_count'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ENGINE_init'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `PKCS12_parse'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSLeay'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `ASN1_STRING_print'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `SSL_get_peer_cert_chain'
/opt/conda/envs/_build/lib/libdf.so: undefined reference to `jpeg_read_scanlines@LIBJPEG_9.0'
/opt/conda/envs/_build/lib/libcurl.so: undefined reference to `UI_set_result'
collect2: error: ld returned 1 exit status
make[2]: *** [nf_test/f90tst_fill2] Error 1
make[1]: *** [nf_test/CMakeFiles/f90tst_fill2.dir/all] Error 2
make: *** [all] Error 2
Command failed: /bin/bash -x -e /recipe_root/build.sh
```

OS X:

``` shell
cc: error: unrecognized command line option ‘-fmacro-backtrace-limit=0’
make[2]: *** [fortran/CMakeFiles/netcdff.dir/nf_lib.c.o] Error 1
make[1]: *** [fortran/CMakeFiles/netcdff.dir/all] Error 2
make: *** [all] Error 2
```

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -1,10 +1,20 @@
 #!/bin/bash
 
+if [[ $(uname) == Darwin ]]; then
+  export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+elif [[ $(uname) == Linux ]]; then
+  export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+export CFLAGS="$CFLAGS -fPIC -I$PREFIX/include"
+
 # Build static.
 cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
       -D CMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
       -D BUILD_SHARED_LIBS=OFF \
       $SRC_DIR
+
 make
 ctest
 make install

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -10,9 +10,24 @@ export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
 export CFLAGS="$CFLAGS -fPIC -I$PREFIX/include"
 
 # Build static.
+mkdir build_static && cd build_static
 cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
       -D CMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
       -D BUILD_SHARED_LIBS=OFF \
+      $SRC_DIR
+
+make
+# ctest  # Run only for the shared lib build to save time.
+make install
+
+make clean
+
+cd ..
+mkdir build_shared && cd build_shared
+# Build shared.
+cmake -D CMAKE_INSTALL_PREFIX=$PREFIX \
+      -D CMAKE_INSTALL_LIBDIR:PATH=$PREFIX/lib \
+      -D BUILD_SHARED_LIBS=ON \
       $SRC_DIR
 
 make

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -28,8 +28,10 @@ requirements:
         - isl 0.12.*
 
 test:
+    requires:
+        - python {{ environ['PY_VER'] + '*' }}  # [win]
     commands:
-        - nf-config --all
+        #- nf-config --all  # FIXME: no nf-config for cmake
         - conda inspect linkages -n _test netcdf-fortran  # [linux]
 
 about:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -32,6 +32,7 @@ test:
         - python {{ environ['PY_VER'] + '*' }}  # [win]
     commands:
         #- nf-config --all  # FIXME: no nf-config for cmake
+        - test -f ${PREFIX}/lib/libnetcdf.a  # [unix]
         - conda inspect linkages -n _test netcdf-fortran  # [linux]
 
 about:

--- a/recipe/run_test.py
+++ b/recipe/run_test.py
@@ -1,0 +1,18 @@
+# Load the libraries using ctypes.
+import os
+import sys
+import ctypes
+
+platform = sys.platform
+
+if platform.startswith('linux'):
+    path = os.path.join(sys.prefix, 'lib', 'libnetcdff.so')
+    lib = ctypes.CDLL(path)
+elif platform == 'darwin':
+    path = os.path.join(sys.prefix, 'lib', 'libnetcdff.dylib')
+    lib = ctypes.CDLL(path)
+elif platform == 'win32':
+    path = os.path.join(sys.prefix, 'Library', 'bin', 'libnetcdff.dll')
+    lib = ctypes.CDLL(path)
+else:
+    raise ValueError('Unrecognized platform: {}'.format(platform))

